src/main/resources/icons/web-assembly-file.svg

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
the image is still too big for using inside as a file type image (maybe the problem here is that pointers != pixels)

Reply:
Can you explain the problem in details? [Here](https://jetbrains.org/intellij/sdk/docs/reference_guide/work_with_icons_and_images.html#image-formats) is all necessary sizes of a plugin icon. Now I have only one size (16x16) -- maybe it's suitable not for everything.